schemas/computation.schema.tpl.json

```diff
@@ -0,0 +1,84 @@
+{
+    "_type": "https://openminds.ebrains.eu/computation/Computation",
+    "_extends": "/core/research/activity.schema.tpl.json",
+    "required": [
+        "input",
+        "output",
+        "protocol",
```

Comment:
I guess "protocol" needs to be now "launchConfiguration"?

Reply:
yes, thanks. Fixed.